ranger2/aws/ranger_asg/ranger_policy.py

```diff
@@ -0,0 +1,161 @@
+import requests
+import json, sys
```

Comment:
there are quite a lot of tpl and py files here. Have we seen what happens when update the terraform configuration? say you perform terraform apply - everything runs fine and deploys. next you want to change something, say some subnet somewhere, will these policy py files run again? if so, will it cause duplicate policies or inconsistencies?

Reply:
No py file will not rerun. Script checks for exist condition before creating.